### PR TITLE
Remove paper-button and paper-card usage

### PR DESCRIPTION
**Pull request by ebidel**
R: @jeffposnick @jpchase @beaufortfrancois @RByers 

Part of #352

### CODE REVIEW

File: templates/features.html [hunk 1]
```diff
@@ -103,6 +97,15 @@
   search.value = location.hash.substr(1);
 }
 
+// $('.actionlinks .blue-button').addEventListener('click', function(e) {
```
Comment (beaufortfrancois) [inline templates/features.html]: Why keep the comment there?

**Comment by beaufortfrancois** [comment]
Should we remove `paper-button` as well in https://github.com/GoogleChrome/chromium-dashboard/blob/nobutton/static/sass/features/features.scss#L36?

LGTM with nit.

File: templates/features.html (hunk 1 above)
Comment (ebidel) [inline templates/features.html]: Done

**Comment by ebidel** [comment]
Done x2

**Pull request merged**